readme.md

```diff
@@ -86,6 +86,52 @@ Run `np` without arguments to launch the interactive UI that guides you through
 <img src="screenshot-ui.png" width="1290">
 
 
+## Config
+
+`np` can be configured both locally and globally. When using the global `np` binary, you can configure any of the CLI flags in either a `.np-config.js` or `.np-config.json` file in the home directory. When using the local `np` binary, for example, in a `npm run` script, you can configure `np` by setting the flags in either a top-level `np` field in `package.json` or in a `.np-config.js` or `.np-config.json` file in the project directory.
+
+Currently, these are the flags you can configure:
+
+- `anyBranch` - Allow publishing from any branch (`false` by default).
```

Comment:
I added default values to all options, but I feel like in some fields it's a bit redundant (such as this one).

Reply:
Why is it redundant?

Reply:
I guess it only seems redundant to me as I'm used to the default values of the CLI options.